package.json

```diff
@@ -66,15 +66,14 @@
     "install:wpcli": "(test -x bin/wp-cli.phar || curl -Ls https://raw.githubusercontent.com/wp-cli/builds/gh-pages/phar/wp-cli.phar -o bin/wp-cli.phar && chmod +x bin/wp-cli.phar) && bin/wp-cli.phar --info",
     "prewp": "npm run install:wpcli",
     "wp": "php -d memory_limit=4G bin/wp-cli.phar",
-    "makepot:android": "npm run wp -- i18n make-pot ./gutenberg/packages --include=*.native.js,*.android.js --exclude=test/* --subtract=./gutenberg.pot --ignore-domain gutenberg-android.pot",
-    "makepot:ios": "npm run wp -- i18n make-pot ./gutenberg/packages --include=*.native.js,*.ios.js --exclude=test/* --subtract=./gutenberg.pot --ignore-domain gutenberg-ios.pot",
+    "makepot": "npm run wp -- i18n make-pot ./jetpack/extensions --include=*.native.js,*.$PLATFORM.js --exclude=test/* --subtract=./gutenberg.pot --ignore-domain jetpack-$PLATFORM.pot && npm run wp -- i18n make-pot ./gutenberg/packages --include=*.native.js,*.$PLATFORM.js --exclude=test/* --subtract=./gutenberg.pot --merge=./jetpack-$PLATFORM.pot --ignore-domain gutenberg-$PLATFORM.pot ",
     "premakepot:gutenberg": "npm run clean:gutenberg && npm run build:gutenberg",
     "makepot:gutenberg": "npm run wp -- i18n make-pot ./gutenberg/build --ignore-domain gutenberg.pot",
     "postmakepot:gutenberg": "npm run clean:gutenberg",
     "pregenstrings": "test -f gutenberg.pot || npm run makepot:gutenberg",
```

Comment:
I am not sure if there's anything similar we need to do for jetpack. The Jetpack submodule we have checked out [has a `languages/` folder that includes all kinds of good stuff, including a `jetpack.pot` file](https://github.com/Automattic/jetpack/tree/f6f313d268c336491c8bca1afcf3e5d3663597ce/languages), but [that `languages/` folder has been removed on `master`](https://github.com/Automattic/jetpack/pull/16518). Basically, there is more work to be done here to either (a) verify that we do not need to do anything more to get all the relevant jetpack strings; or (b) run any necessary jetpack builds as part of the bundling process.